### ISSUE TITLE
Cannot call method "map" of undefined. (line 179, file "FirestoreDocument.gs")

### ISSUE THREAD

**Issue by adyscorpius** [grahamearley/FirestoreGoogleAppsScript#30]
![image](https://user-images.githubusercontent.com/1511089/41044436-e85ae8c6-69c3-11e8-90d5-a734a3e29fd0.png)

I'm getting the above error where trying to query into a nested collection. Anyone else getting this error and how to resolve?

```javascript
var runningProcesses = firestore.query("collection/doument/collection").where("key1","==","value").execute();
Logger.log(runningProcesses.length);
```

**Comment by grahamearley** [comment]
Hey @adyscorpius, thanks for reporting this. Could you post some example data that you are querying? What do those documents look like?

**Comment by adyscorpius** [comment]
![image](https://user-images.githubusercontent.com/1511089/41057984-38b54b00-69e6-11e8-8626-fb8a746394a0.png)

That is the structure inside the document. There are some arrays of objects, which probably fail the unwrap map on line 179.

**Comment by grahamearley** [comment]
Fixed with #31. This use case now works in [library version 17](https://github.com/grahamearley/FirestoreGoogleAppsScript/releases/tag/v17)!
